### PR TITLE
Feedback widget component

### PR DESCRIPTION
**Pull request by punalsc**
Hi @gtvj ,
Submitted a new PR, under a new branch.

Could you please have a look.

Thanks

### CODE REVIEW

File: components/details_feedback/child_components/checkbox/details_feedback_widget_checkbox.react.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+          id={id}
+          value={id}
+          onClick={onClick}
+          autoFocus={autoFocus}
+        />
+        <label for={id}>{labelText}</label>
+      </div>
+    );
+  }
+  static propTypes = {
```
Comment (gtvj) [inline components/details_feedback/child_components/checkbox/details_feedback_widget_checkbox.react.js]: Should `autofocus` have a default prop type too?

File: components/details_feedback/child_components/comment/details_feedback_comment.react.js [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+};
+
+Comment.defaultProps = {
+  id: null,
+  commentLabel: null,
+  onChange: null,
+  autoFocus: null
+};
+
+Comment.propTypes = {
```
Comment (gtvj) [inline components/details_feedback/child_components/comment/details_feedback_comment.react.js]: Do we need a proptype for `commentWarning`

File: components/details_feedback/child_components/checkbox/details_feedback_widget_checkbox.react.js (hunk 1 above)
Comment (punalsc) [reply components/details_feedback/child_components/checkbox/details_feedback_widget_checkbox.react.js]: fixed

File: components/details_feedback/child_components/comment/details_feedback_comment.react.js (hunk 2 above)
Comment (punalsc) [reply components/details_feedback/child_components/comment/details_feedback_comment.react.js]: fixed

**Pull request merged**